style.css

```diff
@@ -258,6 +258,40 @@ a {
   margin-bottom: 59px;
 }
 
+/* .mobile-menu{
+  height: 100vh;
+  width: 100%;
+  margin-top: 80px;
+} */
+
+.mobile-menu ul {
```

Comment:
- [ ] Please, kindly correct the fonts in your mobile menu.  Should be Inter instead of Times New Roman:

![image](https://user-images.githubusercontent.com/57421823/156113491-9e65a709-fc17-4ebd-88f1-bb5836bcad29.png)

That way, your mobile menu will look like the figma.

Reply:
I have added the inter font family